test/rpc/test_rpc_class.py

```diff
@@ -108,15 +66,17 @@ def test_get_deprecated_class(class_hash):
     """
     Test get contract class
     """
-    resp = rpc_call(
-        "starknet_getClass", params={"block_id": "latest", "class_hash": class_hash}
-    )
-    contract_class = resp["result"]
+    with open(ABI_PATH, mode="r", encoding="utf-8") as expected_abi:
+        expected_abi_json = json.loads(expected_abi.read())
```

Comment:
Here and in other `with open` blocks, I think it's ok to just leave the first line indented (i.e. the only one actually depending on the resource being handled by the with-block). In this case L70.

Reply:
Addressed here: [0a8854e](https://github.com/0xSpaceShard/starknet-devnet/pull/494/commits/0a8854ee426523fd6adaa3750a599523a62f3463)